### ISSUE TITLE
Verbum:  Focus State on Comment Block missing styling 

### ISSUE THREAD

**Issue by kraftbj** [Automattic/wp-calypso#83680]
### Quick summary

Hi there!
I'd like to flag an A11y issue I think I have picked up with the comment block.

If you click into the comment block, and press tab, I would expect there to be a focus state where you can see what element is selected.

This seems to be missing at the moment for the account/gravatar link, the submit comment link (and possibly others - it's hard to tell).

You can test this on:

    https://ma.tt/2023/10/texts-joins-automattic/
    https://writerbeware.blog/2023/10/20/why-writer-beware-doesnt-recommend-or-endorse-agents-or-publishers/
    https://wordpress.com/blog/2023/10/18/introducing-command-palette/ 

On all three examples (on different hosts/platforms) - the issue is there.

Here is a video recording:

https://github.com/Automattic/wp-calypso/assets/27249804/a89e9b6e-3d07-4d71-b61c-b2ee3df58f1a

I press tab 4x times, and it only shows on the 4th link, when the focus state for "Get Started" is selected, in the block below.

### Steps to reproduce

see video above

### What you expected to happen

..

### What actually happened

..

### Impact

All

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Simple, Atomic, Self-hosted

### Logs or notes

copying it over from https://core.trac.wordpress.org/ticket/59747

**Comment by cuemarie** [comment]
📌 REPRODUCTION RESULTS
- Tested on https://wordpress.com/blog/ – Replicated 

📌 FINDINGS/SCREENSHOTS/VIDEO
I can replicate this somewhat – inconsistent results:

Firefox – User Settings Open
- _most items show focus state, except:_
     - _log out link_
     - _gravatar/user settings button_

https://github.com/Automattic/wp-calypso/assets/27249804/420d729d-29fd-4642-abc6-279f4bcac3a1

Firefox – User Settings Closed 
- _3 tabs to get to Reply_

https://github.com/Automattic/wp-calypso/assets/27249804/0a7b5b76-905e-49a3-9f30-50ffc750ab94


Chrome
- _most items show focus state, except:
     - _gravatar/user settings_

https://github.com/Automattic/wp-calypso/assets/27249804/ecc7ce0d-4d3d-4e21-bd8f-9073081ca66d

Chrome – User Settings Closed
- _4 tabs to get to Reply, and you can see that the Logout link is highlighted_

https://github.com/Automattic/wp-calypso/assets/27249804/e059cf15-e0d1-496f-b188-654d4c264e38


📌 ACTIONS
- Triaged

**Comment by cuemarie** [comment]
FYI @Automattic/vertex

**Comment by alshakero** [comment]
Hi everyone! Closing as fixed on Nov 2nd.

Fix was here: D127560-code
